src/euphorie/client/browser/session.py

```diff
@@ -222,6 +222,11 @@ def update(self):
             )
 
 
+class MoreInfo(SessionMixin):
```

Comment:
Just use the `SessionMixin` class in the zcml. No need for an extra class right now